tfjs-backend-webgpu/src/webgpu_util.ts

```diff
@@ -168,10 +168,9 @@ export function ArrayBufferToTypedArray(data: ArrayBuffer, dtype: DataType) {
 }
 
 export function isWebGPUSupported(): boolean {
-  if (!navigator.gpu) {
-    return false;
-  }
-  return true;
+  return ((typeof window !== 'undefined') ||
+    //@ts-ignore
+    (typeof WorkerGlobalScope !== 'undefined')) && navigator.gpu
```

Comment:
@crowlKats Please find the bot failure detail [here](https://console.cloud.google.com/cloud-build/builds/35fdfde3-10b9-478a-b3a1-37e1ba888f33;step=2?project=834911136599).
```
Step #4 - "tslint": /workspace/tfjs-backend-webgpu/src/webgpu_util.ts:173:65
Step #4 - "tslint": ERROR: 173:65  semicolon  Missing semicolon
```

Reply:
It seems that you can't directly return `navigator.gpu` which's value is not a bool.
```
Step #2 - "run-cloudbuild": Step #5 - "bazel-tests": tfjs-backend-webgpu/src/webgpu_util.ts:171:3 - error TS2322: Type 'GPU' is not assignable to type 'boolean'.
Step #2 - "run-cloudbuild": Step #5 - "bazel-tests": 
Step #2 - "run-cloudbuild": Step #5 - "bazel-tests": 171   return ((typeof window !== 'undefined') ||
Step #2 - "run-cloudbuild": Step #5 - "bazel-tests":       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Step #2 - "run-cloudbuild": Step #5 - "bazel-tests": 172     //@ts-ignore
Step #2 - "run-cloudbuild": Step #5 - "bazel-tests":     ~~~~~~~~~~~~~~~~
Step #2 - "run-cloudbuild": Step #5 - "bazel-tests": 173     (typeof WorkerGlobalScope !== 'undefined')) && navigator.gpu;

Reply:
And you can use `yarn && yarn build` under tfjs-backend-webgpu to see if everything is good.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.